PADC-FP-TVGuide-Myanmar/app/src/main/java/com/padc/tvguide/activities/ProgramDetailActivity.java

```diff
@@ -0,0 +1,51 @@
+package com.padc.tvguide.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.design.widget.FloatingActionButton;
+import android.support.design.widget.Snackbar;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.CardView;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+
+import com.padc.tvguide.R;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+
+public class ProgramDetailActivity extends AppCompatActivity {
+
+    @BindView(R.id.cv_parent)
+    CardView cvParent;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_program_detail);
+        ButterKnife.bind(this);
+
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+
+        FloatingActionButton fab = (FloatingActionButton) findViewById(R.id.fab_search);
```

Comment:
You can also use @BindView for FAB, if FAB isn't necessary, remove it from the activity layout completely.